parts/linux/cloud-init/artifacts/mariner/cse_install_mariner.sh

```diff
@@ -25,6 +25,47 @@ installDeps() {
     fi
 }
 
+installKataDeps() {
+    if [[ $OS_VERSION == "2.0" ]]; then
```

Comment:
nit: 
It might good to use $OS_VERSION != "1.0", as we will likely want this code to continue to run in future major version releases and this approach could lead to less maintenance.

Reply:
Good point! Updated the branch condition as suggested.